src/include/gpopt/gpdbwrappers.h

```diff
@@ -303,6 +303,9 @@ namespace gpdb {
 	// find the oid of the root partition given partition oid belongs to
 	Oid OidRootPartition(Oid oid);
 	
+	// find the number of partitions for a given root relation
+	int INumberPartitions(Oid oid);
+
```

Comment:
how is it different to 
{code}
gpos::ULONG UlLeafPartitions(Oid oidRelation);
{code}

Can we get rid one of them? Also can we make the return of the gpos::ULONG?